src/test/java/organice/model/person/TissuetypeTest.java

```diff
@@ -0,0 +1,68 @@
+package organice.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static organice.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TissuetypeTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Tissuetype(null));
+    }
+
+    @Test
+    public void constructor_invalidTissuetype_throwsIllegalArgumentException() {
+        String invalidTissuetype = "";
+        assertThrows(IllegalArgumentException.class, () -> new Tissuetype(invalidTissuetype));
+    }
+
+    @Test
+    public void isValidTissuetype() {
+        // null tissuetype
+        assertThrows(NullPointerException.class, () -> Tissuetype.isValidTissuetype(null));
+
+        // invalid tissuetype
+        assertFalse(Tissuetype.isValidTissuetype("")); // empty string
+        assertFalse(Tissuetype.isValidTissuetype(" ")); // spaces only
+        assertFalse(Tissuetype.isValidTissuetype("a,b,c,d,e,f")); // wrong tissuetype
```

Comment:
Perhaps would be better to say `\\ tissue types must be Integers from 1..12`. The dots are intentional btw. `..` means range inclusive, `...` means range exclusive